### ISSUE TITLE
Fails to start a on recent Debian on Vte.Terminal.set_word_chars

### ISSUE THREAD

**Issue by sylvestre** [Guake/guake#432]
With python-vte Debian package 0.28.2-5
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib/python2.7/dist-packages/guake/main.py", line 216, in <module>
    exec_main()
  File "/usr/lib/python2.7/dist-packages/guake/main.py", line 212, in exec_main
    if not main():
  File "/usr/lib/python2.7/dist-packages/guake/main.py", line 127, in main
    instance = Guake()
  File "/usr/lib/python2.7/dist-packages/guake/guake_app.py", line 263, in __init__
    self.add_tab()
  File "/usr/lib/python2.7/dist-packages/guake/guake_app.py", line 1139, in add_tab
    box = GuakeTerminalBox()
  File "/usr/lib/python2.7/dist-packages/guake/terminal.py", line 244, in __init__
    self.terminal = GuakeTerminal()
  File "/usr/lib/python2.7/dist-packages/guake/terminal.py", line 79, in __init__
    self.configure_terminal()
  File "/usr/lib/python2.7/dist-packages/guake/terminal.py", line 93, in configure_terminal
    self.set_word_chars(word_chars)
TypeError: Vte.Terminal.set_word_chars() argument 1 must be string, not None
```

**Comment by ZiTAL** [comment]
same in ubuntu 15.10

**Comment by Stibbons** [comment]
should be fixed on 0.8.x
